otterdog/eclipse-kuksa.jsonnet

```diff
@@ -53,12 +66,7 @@ orgs.newOrg('eclipse-kuksa') {
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
       branch_protection_rules: [
-        orgs.newBranchProtectionRule('main') {
-          required_approving_review_count: 0,
-          requires_linear_history: true,
-          requires_status_checks: false,
-          requires_strict_status_checks: true,
-        },
+        kuksa_default_branch_protection_rule('main')
```

Comment:
This include some functional differences, like removing requirement for linear history